runtime/vm/jvminit.c

```diff
@@ -663,6 +663,11 @@ freeJavaVM(J9JavaVM * vm)
 		vm->cifArgumentTypesCache = NULL;
 	}
 
+	/* Delete the layout string hashtable if exists. */
+	if (NULL != vm->layoutStrFFITypeTable) {
+		releaseLayoutStrFFITypeTable(vm->layoutStrFFITypeTable);
```

Comment:
Please clear the pointer (like line 663):
```c
		vm->layoutStrFFITypeTable = NULL;
```

Reply:
Fixed.